src/app/work-item/work-item.service.ts

```diff
@@ -16,25 +16,46 @@ export class WorkItemService {
   private workItemTypeUrl = process.env.API_URL + 'workitemtypes';
   private availableStates: DropdownOption[] = [];
   public workItemTypes: WorkItemType[] = [];
+  private isSearching: Boolean = false;
+  private nextLinks: Object = {};
```

Comment:
Should probably be called just 'links'. nextLinks is just one of the many possible links. And not all links would be related to pagination.